docs/standard/microservices-architecture/multi-container-microservice-net-applications/data-driven-crud-microservice.md

```diff
@@ -240,14 +247,13 @@ From your docker-compose.yml or docker-compose.override.yml files, you can initi
 catalog.api:
   environment:
     - ConnectionString=Server=sql.data;Database=Microsoft.eShopOnContainers.Services.CatalogDb;User Id=sa;Password=Pass@word
-    - ExternalCatalogBaseUrl=http://10.0.75.1:5101
-    #- ExternalCatalogBaseUrl=http://dockerhoststaging.westus.cloudapp.azure.com:5101
-  
+    # Additional environment variables for this service
   ports:
-    - "5101:5101"
+    - "5101:80"
 ```
 
-The docker-compose.yml files at the solution level are not only more flexible than configuration files at the project or microservice level, but also more secure. Consider that the Docker images that you build per microservice do not contain the docker-compose.yml files, only binary files and configuration files for each microservice, including the Dockerfile. But the docker-compose.yml file is not deployed along with your application; it is used only at deployment time. Therefore, placing environment variables values in those docker-compose.yml files (even without encrypting the values) is more secure than placing those values in regular .NET configuration files that are deployed with your code.
+The docker-compose.yml files at the solution level are not only more flexible than configuration files at the project or microservice level, but also more secure. Consider that the Docker images that you build per microservice do not contain the docker-compose.yml files, only binary files and configuration files for each microservice, including the Dockerfile. But the docker-compose.yml and docker-compose.override.yml files are not included into the Docker image, not deployed along with your application; it is used only at deployment time. 
```

Comment:
I don't agree. The docker-compose yml files are still text, in the source of your application, and might be in a version control system. 

For security, shouldn't this information be part of a secure deployment portal, not in source?

Reply:
You are right. In reality, what's more secure is the usage of environment variables that can come from the deployment environment, like providing specific env-vars values from within the VSTS deployment pipelines. I'll change that.